src/DeferredStream.php

```diff
@@ -15,6 +15,13 @@ class DeferredStream
      */
     public static function create(ParserInterface $parser)
     {
+        if ($parser->isDone()) {
+            return \React\Promise\resolve([
```

Comment:
Actually I think throwing an exception would make more sense here. Failing silency is pretty nasty